lib/shared/src/conversion.rs

```diff
@@ -213,6 +213,7 @@ const TIMESTAMP_TZ_FORMATS: &[&str] = &[
     "%a %d %b %T %Z %Y",  // `date` command output
     "%a %d %b %T %z %Y",  // `date` command output, numeric TZ
     "%a %d %b %T %#z %Y", // `date` command output, numeric TZ
+    "%d/%b/%Y:%T %z",     // Common Log
```

Comment:
It would be good to add a line to the `parse_timestamp_auto_tz_env` test below to match this format.

Reply:
Good catch 👍 done in 2bab586.